utils/config.go

```diff
@@ -395,3 +441,36 @@ func (config *Config) evaluateCertificate() error {
 	config.IsHost = false
 	return nil
 }
+
+// GetConfigFilePath returns concerto configuration path file
+func GetConfigFilePath(fileFilter string) string {
```

Comment:
This function does not return the ConfigFilePath, but rather the default value for a file path in the configuration: it should be called something similar to GetDefaultFilePath 
You have a string parameter on which you do a switch to determine which kind of file path you have to return. I think that having different GetDefaultXXXFilePath functions, one for every different case would be better.